balta/src/main/scala/za/co/absa/db/balta/classes/DBFunction.scala

```diff
@@ -45,6 +45,24 @@ sealed abstract class DBFunction private(functionName: String,
     s"SELECT * FROM $functionName($paramsLine) $orderBy"
   }
 
+  /**
+   * Executes the function without caring about the result. The goal is the side-effect of the function.
+   * @param connection  - the database connection
+   */
+  def perform()(implicit connection: DBConnection): Unit = {
+    execute("")(_ => ())
+  }
+
+  /**
+   * Executes the function without any verification procedure. It instantiates the function result(s) and returns them in
+   * a list.
+   * @param orderBy     - the clause how to order the function result, if empty, default ordering is preserved
+   * @param connection  - the database connection
+   */
+  def getResult(orderBy: String = "")(implicit connection: DBConnection): List[QueryResultRow] = {
+    execute("")(_.toList)
```

Comment:
Here is not used input parameter 'orderBy' in method body.
See suggestion from our teams chat:

```suggestion
    execute(orderBy)(_.toList)
```